ipinfo/cmd_randip.go

```diff
@@ -53,7 +53,7 @@ Description:
 Options:
   --help, -h
     show help.
-  --count, -n 
+  --num, -n 
```

Comment:
I think we also need to update auto-completion. Just do a `grep` to find all occurrences.